### DISCUSSION TITLE
Difficulty tweaks for Arcade Mode

### DISCUSSION THREAD

**Discussion by Outrunner24** [emoose/OutRun2006Tweaks#256] [Suggestions]
### Describe the feature you'd like to see added

Arcade Mode is just too easy and boring, with only the last stage offering some kind of challenge. Would it be possible to tweak some settings like the original OutRun? For example traffic density, clock speed etc. Also, having the choice of night/day or reverse tracks would be nice, but I think we need a complete editor for that.

Thanks anyway for the great work!

**Comment by emoose** [comment]
There is a setting in games options for arcade difficulty, did you try changing that?
I think even on hardest it might still be a lot easier than actual 2SP though, wonder how much we could change about that.

**Comment by Outrunner24** [comment]
> There is a setting in games options for arcade difficulty, did you try changing that?

Yeah, the only thing that changes is the last couple stages. Having more traffic or a faster clock since the beginning (like in the original) would be better, but they're probably hard coded into the stage logic.

**Comment by Tenome** [comment]
>Also, having the choice of night/day or reverse tracks would be nice, but I think we need a complete editor for that.

Howard's FXT Course Editor does exactly that fyi. I wonder if forcing night/sunset onto tracks that don't have it would be possible though, by loading the skybox/lightbox from the sunset/night levels.

**Comment by Outrunner24** [comment]
> Howard's FXT Course Editor does exactly that fyi. 

I know, but not having to switch mods would be better for me. A .INI flag to reverse tracks, for example, or increase traffic, but I know it's not easy to do.

**Comment by k0pp0** [comment]
Is the fxt mod maker aware of emoose's work? Assuming they are but maybe we could ask politely? 

A gui for the ini is being worked on at the moment

**Comment by k0pp0** [comment]
https://discord.com/channels/1259784325394858058/1259784325394858061/1275902361369317397
